server/run_tests_and_coverage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+docker compose -f docker-compose-test.yml up --detach
+
+sleep 3
+
+cd ./lomas_server
+
+# On admin_database: "mongodb" and dataset_store: "LRU_cache" and production mode
+export LOMAS_TEST_MONGO_INTEGRATION=1
+coverage run --source=. -m unittest discover -s .
+ret=$?
+
+# On admin_database: "yaml" and dataset_store: "basic" and developer mode
+export LOMAS_TEST_MONGO_INTEGRATION=0
+coverage run -a --source=. -m unittest discover -s .
+ret=$?
```

Comment:
You are overwritting the results of the first tests here. You might want "and" both.

Reply:
you mean for the coverage ? the -a should append to the coverage file.

do you mean for ret=$?

Reply:
ah yes for ret, yes i should try to 'and' it indeed

Reply:
if it is not possible, i can also let the tests separately in github actions as before and do another one just to check the coverage

Reply:
nice catch ^^